embd.go

```diff
@@ -30,6 +30,7 @@ import (
 	"flag"
 	"fmt"
 	"io"
+	"io/ioutil"
 	"os"
 	"path/filepath"
 	"regexp"
```

Comment:
1. Please change the version info on line 25 to 1.1 or 2.0 and update the date. Also, please change this comment into a `const version = "embd-go version ... https://..."` just below imports (above flags), and print it with Fprintln just below the Fprintf in flag.Usage.
2. You're welcome to add a second copyright notice line with your name/nickname and email below line 4.